packages/rich-text-editor/src/styles/vaadin-rich-text-editor-base-styles.js

```diff
@@ -35,13 +32,24 @@ const base = css`
     display: none;
   }
 
+  .vaadin-field-container {
+    display: flex;
+    flex-direction: column;
```

Comment:
Generally it's better not to rely on this element being present, as we changed it to use `display: contents` in the baseline alignment PR https://github.com/vaadin/web-components/pull/10320. @jouni FYI, what should we do about this?

Reply:
Yeah, I think I might leave out this wrapper element, and set the layout on the host.